server/src/main/java/org/apache/druid/indexing/overlord/SegmentPublishResult.java

```diff
@@ -34,22 +34,14 @@
 import java.util.Set;
 
 /**
- * Result of an operation that attempts to publish segments. Indicates the set of segments actually published
- * and whether or not the transaction was a success.
- *
- * If "success" is false then the segments set will be empty.
- *
- * It's possible for the segments set to be empty even if "success" is true, since the segments set only
- * includes segments actually published as part of the transaction. The requested segments could have been
- * published by a different transaction (e.g. in the case of replica sets) and this one would still succeed.
+ * Result of a segment publish operation.
  */
 public class SegmentPublishResult
 {
+  private final boolean canRetry;
   private final Set<DataSegment> segments;
   private final boolean success;
```

Comment:
Super nit cause you love beautiful code. 
Should line 41 go to line 44 after `private final boolean success;` the constructor args and the class fields are in the same order.

Reply:
Sure, will do 😄